shiki.config.mjs

```diff
@@ -40,7 +40,7 @@ export default {
     ...diffLanguage,
     ...cLanguage,
     ...cPlusPlusLanguage,
-    ...httpLanguage,
     ...coffeeScriptLanguage,
+    { ...javaScriptLanguage[0], aliases: ['mjs', 'cjs', 'js'] },
```

Comment:
Was the reorganisation needed? 🤔

Reply:
For some reason, yes. When `javaScriptLanguage` was first, it caused the parser to throw an error that `mjs` was not found. :thinking: I'm not sure why, maybe one of the other languages overwrote it? I can look into it some more.